src/SFML.Graphics/Font.cs

```diff
@@ -57,7 +59,23 @@ public Font(Stream stream) : base(IntPtr.Zero)
         /// <param name="bytes">Byte array containing the file contents</param>
         /// <exception cref="LoadingFailedException" />
         ////////////////////////////////////////////////////////////
-        public Font(byte[] bytes) : this(new MemoryStream(bytes)) { }
+        public Font(byte[] bytes) :
+            base(IntPtr.Zero)
+        {
+            GCHandle pin = GCHandle.Alloc(bytes, GCHandleType.Pinned);
+            try
+            {
+                CPointer = sfFont_createFromMemory(pin.AddrOfPinnedObject(), (UIntPtr)bytes.Length);
+            }
+            finally
+            {
+                pin.Free();
+            }
+            if (CPointer == IntPtr.Zero)
+            {
+                throw new LoadingFailedException("font");
+            }

```

Comment:
This is change from createFromStream to createFromMemory do anything for us?

Reply:
This way the data is read directly from memory instead of having to make a `MemoryStream`, `StreamAdaptor` and associated callback stuff. It's also more consistent with the other places in the code that have `createFromStream` and `createFromMemory`.